app/styles/components/_detailview.scss

```diff
@@ -120,7 +123,7 @@
 
   .detail-header {
     @include span-columns(12 of 12);
-    background-color: $background-grey;
+    background-color:#edecec;
```

Comment:
There should be no color definitions in any file but variables.scss.  Otherwise it is too easy to loose track of the colors we are using.